plenum/common/constants.py

```diff
@@ -79,6 +79,7 @@
 IDENTIFIER = "identifier"
 TARGET_NYM = "dest"
 DATA = "data"
+META = "meta"
```

Comment:
Although this is fine to have this constant in Plenum, do we really need it here if it's not used? If it's used in Node, then it can be defined there.